internal/guest/bridge/bridge_v2.go

```diff
@@ -368,13 +368,9 @@ func (b *Bridge) getPropertiesV2(r *Request) (_ RequestResponse, err error) {
 		}
 	}
 
-	propertyJSON := []byte("{}")
-	if properties != nil {
-		var err error
-		propertyJSON, err = json.Marshal(properties)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to unmarshal JSON in message \"%+v\"", properties)
-		}
+	propertyJSON, err := json.Marshal(properties)
```

Comment:
huh, neat. didn't know that `json.Marshal` on `nil` will return `{}`

Reply:
i didnt know that either, the linter flagged it cause of the `properties := &prot.PropertiesV2{}` line above (L333), so the branch is always taken